### PR TITLE
refactor: add prettier check action

### PR DESCRIPTION
**Pull request by wadjih-bencheikh18**
closes : https://github.com/zakodium/react-plot-docs/issues/16

### CODE REVIEW

**Comment by wadjih-bencheikh18** [comment]
@stropitek do you want check to be also in PR ?

File: demo-loader.webpack.js [hunk 1]
```diff
@@ -61,15 +60,11 @@ Example:
...
-  const codeSandboxImportPath = path
-    .relative(this.context, path.join('src', 'components', 'CodeSandboxer.tsx'))
-    .replaceAll(path.sep, path.posix.sep);
-
   const modifiedSource = `
   ${beforeDefaultExportSource}
   import { useState as __useState__ } from 'react';
   import CodeBlock from '@theme/CodeBlock';
-  import CodeSandboxer from '${codeSandboxImportPath}';
+  import CodeSandboxer from '/src/components/CodeSandboxer.tsx';
```
Comment (wadjih-bencheikh18) [inline demo-loader.webpack.js:67]: @stropitek the solution you merged did'nt work for me, so i found this solution

Comment (wadjih-bencheikh18) [reply demo-loader.webpack.js]: the solution work now so i deleted what i changed

**Comment by stropitek** [comment]
> @stropitek do you want check to be also in PR ?

Yes please

**Comment by targos** [comment]
The check should be in a separate workflow.

**Pull request merged**